ZupIT/ritchie-cli
# Functional Tests Rit Completion

### Description

Functional tests of the command `rit completion`
### How to verify it
You can verify on menu Actions or submit changes on the files listed in workflow  
### Changelog

---------------------------------------------### This pull request generated the following artifacts.
To test the health and quality of this implementation, download the respective binary for your operating system, unzip and directly run the binary like the examples below.
- **Windows** Download the file: **[rit-windows.zip](https://github.com/ZupIT/ritchie-cli/suites/3300171771/artifacts/77126457)** Unzip to some folder like: `C:\home\user\downloads\pr1010` Access the folder: `cd C:\home\user\downloads\pr1010` Directly call the binary: `.\rit.exe --version` or `.\rit.exe name of formula`  - **Linux** Download the file: **[rit-linux.zip](https://github.com/ZupIT/ritchie-cli/suites/3300171771/artifacts/77126455)** Unzip to some folder like: `/home/user/downloads/pr1010` Access the folder: `cd /home/user/downloads/pr1010` Assign execute permission to binary: `chmod +x ./rit` Directly call the binary: `./rit --version` or `./rit name of formula`  - **MacOS** Download the file: **[rit-macos.zip](https://github.com/ZupIT/ritchie-cli/suites/3300171771/artifacts/77126456)** Unzip to some folder like: `/home/user/downloads/pr1010` Access the folder: `cd /home/user/downloads/pr1010` Assign execute permission to binary: `chmod +x ./rit` Directly call the binary: `./rit --version` or `./rit name of formula`> Generated at Thu Jul 22 2021 03:11:07 GMT+0000 (Coordinated Universal Time)
